config/dispatcher/dispatcher.ex

```diff
@@ -75,6 +75,13 @@ defmodule Dispatcher do
     Proxy.forward conn, [], "http://frontend/index.html"
   end
 
+  ###############################################################
+  # login
```

Comment:
nitpicky / optional but I would rename it "mock login" since we will probably add real login routes in the future?

Reply:
everything related to the login will go under this; i'm affraid it turns as a meme but i'll say it again "it's like in the loket" XD